views/signIn.hbs

```diff
@@ -0,0 +1,28 @@
+{{> header}}
```

Comment:
Include doctype and html tag in this file. And create separate partial for head. Header should contain html only for header and navigation.